.typos.toml

```diff
@@ -11,3 +11,4 @@ Typ = "Typ"
 fo = "fo"
 typ = "typ"
 Unknwon = "Unknwon"
+Advices = "Advices"
```

Comment:
The plural of "advice" is "advice"... Same as "sheep" (which multiple of is "sheep", not "advice" 🥸)